Spotify.js

```diff
@@ -151,7 +151,7 @@ class Spotify {
           this.handleRequestError(error)
           if (cb) {
             let retryTimerInSeconds = 5
-            if (typeof error.response.headers["retry-after"] !== "undefined") retryTimerInSeconds = error.response.headers["retry-after"]
+            if (error.response && typeof error.response.headers["retry-after"] !== "undefined") retryTimerInSeconds = error.response.headers["retry-after"]
```

Comment:
So this differs from the suggested fix you mentioned running locally in https://github.com/skuethe/MMM-Spotify/issues/193?!

```js
-            if (typeof error.response.headers["retry-after"] !== "undefined") retryTimerInSeconds = error.response.headers["retry-after"]
+            if (typeof error.response?.headers["retry-after"] !== "undefined") retryTimerInSeconds = error.response.headers["retry-after"]
```
Is there a specific reason why? :)

Reply:
I couldn't find the info I wanted on the supported node versions so I went with this because it's more backwards compatible.